tracee-ebpf/Makefile

```diff
@@ -147,11 +139,15 @@ define docker_builder_make
 	--entrypoint make $(DOCKER_BUILDER) KERN_BLD_PATH=$(DOCKER_BUILDER_KERN_BLD) KERN_SRC_PATH=$(DOCKER_BUILDER_KERN_SRC) $(1)
 endef
 
+.PHONY: mostlyclean
+mostlyclean:
+	-rm -rf $(OUT_BIN) $(bpf_bundle_dir) $(OUT_BPF) $(BPF_BUNDLE)
```

Comment:
Why have two cleans?

Reply:
clean should delete all resources that were created by the makefile but some resources are 1) not changed between builds and 2) take long time to rebuild so it's better to keep them around. For example: libbpf, or the lower docker layers are useful to keep around. I followed the convention described here https://www.gnu.org/software/make/manual/html_node/Goals.html